trading/strategies/buy_and_hold.go

```diff
@@ -1,18 +1,11 @@
 package main
 
-import (
-	"flag"
-
-	"github.com/d-sparks/ace-of-trades/trading"
-	"github.com/d-sparks/ace-of-trades/trading/simulate"
-	"github.com/d-sparks/ace-of-trades/trading/strategies/buy_and_hold"
-)
-
-var ticks = flag.String("ticks", "./data/kaggle/historical_as_ticks.json", "Kaggledata")
-var symbols = flag.String("symbols", "./data/kaggle/historical_stocks.csv", "Stock symbols")
-var output = flag.String("output", "./results", "Results output")
+//var ticks = flag.String("ticks", "./data/kaggle/historical_as_ticks.json", "Kaggledata")
```

Comment:
I'll go ahead and delete trading/strategies and trading/simulate when I'm done implementing buythedip in the Signal/Strategy setup.